implants/lib/eldritch/src/file/mkdir_impl.rs

```diff
@@ -1,8 +1,17 @@
 use anyhow::Result;
 use std::fs;
 
-pub fn mkdir(path: String) -> Result<()> {
-    match fs::create_dir(&path) {
+pub fn mkdir(path: String, parent: Option<bool>) -> Result<()> {
+    let mut parent_val = false;
+    if parent.is_some() {
+        parent_val = parent.unwrap();
+    }
```

Comment:
Maybe:
```rust
if let Some(parent_val) = parent {
     match parent_val {
         True => { create_dir_all(&path) }
         False => { create_dir(&path) }
     }
}
```

Reply:
in order to have default i think we need it like this? (see change)

Reply:
That's fair.